### PR TITLE
Breadcrumbs for Layer Listing

### PR DESCRIPTION
**Pull request by ClockworkOnion**

### CODE REVIEW

**Comment by ClockworkOnion** [comment]
[Breadcrumbs fürs Layerlisting](https://trello.com/c/ePrStiCR/894-breadcrumbs-f%C3%BCrs-layerlisting)

File: angular.json [hunk 1]
```diff
@@ -797,5 +797,8 @@
                 }
             }
         }
+    },
+    "cli": {
+        "analytics": "a023a7a7-330e-4272-8ba5-9c5c0acfd65e"
```
Comment (ChristianBeilschmidt) [inline angular.json:802]: Why did you add this?

Comment (ClockworkOnion) [reply angular.json:802]: I believe this appeared when I had to do npm install again, possibly even before the switch to angular 14 but after one of the major updates to the main branch. I guess I accidentally added it to the commit. I'll delete these lines in the next commit

File: projects/wave-core-new/src/lib/layer-collections/layer-collection-list/layer-collection-list.component.ts [hunk 2]
```diff
@@ -124,8 +124,9 @@ export class LayerCollectionListComponent implements OnInit, AfterViewInit {
 
     select(item: LayerCollectionItemDict): void {
         if (item.type === 'collection') {
-            const collection = item as LayerCollectionDict;
-            this.selectListener(collection.id);
+            this.selectListener(item);
+            // const collection = item as LayerCollectionDict;
```
Comment (michaelmattig) [inline projects/wave-core-new/src/lib/layer-collections/layer-collection-list/layer-collection-list.component.ts:128]: remove

File: projects/wave-core-new/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts [hunk 3]
```diff
@@ -10,47 +10,96 @@ import {CONTEXT_TOKEN, LayerCollectionListComponent} from '../layer-collection-l
...
+    @ViewChild('scrollElement', {read: ElementRef}) public scrollElement!: ElementRef<any>;
+
     constructor() {
         this.setPortal(undefined);
     }
 
+    scrollToRight(): void {
+        setTimeout(() => {
+            // wait until breadcrumbs are re-rendered before scrolling
+            this.scrollElement.nativeElement.scrollLeft += 500;
```
Comment (michaelmattig) [inline projects/wave-core-new/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts:30]: will scrolling 500 always work? Maybe you could calculate the width of the div and set it to the correct value instead.

File: projects/wave-core-new/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts [hunk 4]
```diff
@@ -10,47 +10,96 @@ import {CONTEXT_TOKEN, LayerCollectionListComponent} from '../layer-collection-l
...
+        }, 50);
```
[the remark this answers is not in the document]
Comment (ClockworkOnion) [reply projects/wave-core-new/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts]: It works with timeout of 0, so I used that 4a8a0b13e38c962311cf7b0d4654be79406cd047

File: projects/wave-core-new/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts (hunk 3 above)
Comment (ClockworkOnion) [reply projects/wave-core-new/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts]: I scrolled to the right by the max possible value (maybe a bit of a sledgehammer approach, but it saves having to calculate widths) 4a8a0b13e38c962311cf7b0d4654be79406cd047

**Comment by ClockworkOnion** [comment]
New Pull Request: https://github.com/geo-engine/geoengine-ui/pull/423 contains all changes after file/folder renaming
